reference-architecture/osp-cli/ch4.8.4_configure_docker_storage_nodes.sh

```diff
@@ -0,0 +1,6 @@
+for H in $NODES
+do
+  scp docker-storage-setup ${H}:
+  ssh $H sudo cp docker-storage-setup /etc/sysconfig
+  ssh $H sudo /usr/bin/docker-storage-setup
```

Comment:
I was under the assumption that it would not, as the heat templates do it explicitly.